rotkehlchen/chain/binance_sc/manager.py

```diff
@@ -0,0 +1,42 @@
+from typing import TYPE_CHECKING
+
+from rotkehlchen.chain.evm.manager import EvmManager
+
+from .accountant import BinanceSCAccountingAggregator
+from .decoding.decoder import BinanceSCTransactionDecoder
+from .tokens import BinanceSCTokens
+from .transactions import BinanceSCTransactions
+
+if TYPE_CHECKING:
+
+    from .node_inquirer import BinanceSCInquirer
+
+
+class BinanceSCManager(EvmManager):
+
+    def __init__(
+            self,
+            node_inquirer: 'BinanceSCInquirer',
+    ) -> None:
+        transactions = BinanceSCTransactions(
+            binance_sc_inquirer=node_inquirer,
+            database=node_inquirer.database,
+        )
+        super().__init__(
+            node_inquirer=node_inquirer,
+            transactions=transactions,
+            tokens=BinanceSCTokens(
```

Comment:
we now have the walrus operator

```suggestion
            transactions=(transactions := BinanceSCTransactions(
            binance_sc_inquirer=node_inquirer,
            database=node_inquirer.database,
        )),
            tokens=BinanceSCTokens(
```